src/andromede/main/main.py

```diff
@@ -67,16 +75,46 @@ def main() -> None:
 
     args = parser.parse_args()
 
-    models = input_models(args.model)
-    components = input_components(args.study, models)
-    consistency_check(components.components, models.models)
+    components = None
+    database = None
+
+    if args.study:
+        if args.models or args.component or args.timeseries:
```

Comment:
I think there's a way in ParseArgs to create exclusive groups, i.e., sets of arguments that can't be passed together so you would automatically check for this

Reply:
there is but you can't make an argument excludes with other arguments which are not mutually exclusive